Running in wrapper mode for '/root/tf-hpolib/HPOlib/benchmarks/tf-cnn/spearmint_april2013_mod_23_2016-8-30--16-35-54-425555/jobs/00020001.pb'
Waiting to lock grid......acquired
Released lock on job grid.
Running python job.
Changed into dir /root/tf-hpolib/HPOlib/benchmarks/tf-cnn/spearmint_april2013_mod_23_2016-8-30--16-35-54-425555
[CRITICAL] [18:01:42:HPOlib.cv] args: (20001L, {u'filter_size': array([1]), u'fc_depth': array([274]), u'conv1_depth': array([512]), u'conv2_depth': array([500])}) kwargs: {}
[INFO] [18:01:42:HPOlib.cv] Starting Cross validation
[INFO] [18:01:42:HPOlib.cv] {u'conv2_depth': 500, u'fc_depth': 274, u'conv1_depth': 512, u'filter_size': 1}
[INFO] [18:01:42:HPOlib.cv] Calling command:
python /usr/local/lib/python2.7/dist-packages/HPOlib-0.1.0-py2.7.egg/HPOlib/runsolver_wrapper.py 0 spearmint_april2013_mod 0 0 0 -conv2_depth 500 -fc_depth 274 -conv1_depth 512 -filter_size 1
[INFO] [18:01:42:HPOlib.cv] --------------RUNNING RUNSOLVER_WRAPPER--------------
[INFO] [18:03:40:HPOlib.cv] Result for ParamILS: SAT, 118.392000, 1, 7.520000, 0, python ../convolutional-hpolib.py

[ERROR] [18:03:40:HPOlib.cv] [INFO] [18:01:42:HPOlib.runsolver_wrapper] -----------------------RUNNING RUNSOLVER----------------------------
[INFO] [18:03:40:HPOlib.runsolver_wrapper] Result for ParamILS: SAT, 118.392000, 1, 7.520000, 0, python ../convolutional-hpolib.py

[INFO] [18:03:40:HPOlib.cv] Finished CV
[INFO] [18:03:40:HPOlib.cv] Result: 7.520000
Got result 7.520000
Job file reloaded.
Completed successfully in 118.66 seconds. [7.520000]
Waiting to lock grid......acquired
Released lock on job grid.
